scrum/tools/bin/calc_sp_by_property.py

```diff
@@ -0,0 +1,111 @@
+""" Print stats about story point allocation on the current sprint board
+    By: label, member
+"""
+
+# Set working directry for module imports
+import os
+import sys
+from colorama import Fore, Style
+os.chdir("/home/pocdart/pocdart_documentation/scrum/tools/")
+current = os.path.dirname(os.path.realpath(__file__))
+parent = os.path.dirname(current)
+sys.path.append(parent)
+
+# Import local modules
+from sprint_utils import load_config
+from trello.api import TrelloAPI
```

Comment:
Consider adding an init.py `tests/test_helpers/__init__.py` file to the `trello` directory in order to support importing like this.